src/script/page/RightSidebar/MessageDetails/MessageDetails.tsx

```diff
@@ -103,7 +118,7 @@ const MessageDetails: FC<MessageDetailsProps> = ({
   const teamId = activeConversation.team_id;
   const supportsReceipts = messageSender.isMe && teamId;
 
-  const supportsLikes = useMemo(() => {
+  const supportsReactions = useMemo(() => {
```

Comment:
LEt's remove this `useMemo` it adds unecessary complexity here. memo should only be used for computation intensive operations

Reply:
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/63250054/226327343-64ca37b3-00b0-498a-8fa2-d27156386b5b.png">

Reply:
🙃 nevermind, it's a problem for later then